MangaManager/ExternalSources/MetadataSources/metadata/AniList.py

```diff
@@ -80,23 +108,39 @@ def trim(value):
     def get_cinfo(cls, series_name) -> ComicInfo | None:
         comicinfo = ComicInfo()
         try:
-            content = cls._search_for_manga_title_by_manga_title(series_name, "MANGA", {})
+            content = cls._search_for_manga_title_by_manga_title(series_name.strip(), "MANGA", {})
         except MangaNotFoundError:
-            content = cls.search_for_manga_title_by_manga_title_with_adult(series_name, "MANGA", {})
+            content = cls.search_for_manga_title_by_manga_title_with_adult(series_name.strip(), "MANGA", {})
 
         if content is None:
             return None
         content = content.get("id")
         data = cls._search_details_by_series_id(content, "MANGA", {})
 
         startdate = data.get("startDate")
-        comicinfo.summary = data.get("description").strip()
         comicinfo.day = startdate.get("day")
         comicinfo.month = startdate.get("month")
         comicinfo.year = startdate.get("year")
-        comicinfo.series = data.get("title").get("romaji").strip()
         comicinfo.genre = ", ".join(data.get("genres")).strip()
         comicinfo.web = data.get("siteUrl").strip()
+        if data.get("volumes"):
+            comicinfo.count = data.get("volumes")
+
+        # Title (Series & LocalizedSeries)
+        title_english = data.get("title").get("english")
+        if cls.aniList_setting.get(AniListSetting.SeriesTitleLanguage) == 'True':
```

Comment:
I would rewrite this so that you pull them into 2 variables then do an if else on how to set them. It will be much easier to understand.